### PR TITLE
Replaced images that wouldn't load, added source

### PR DESCRIPTION
**Pull request by ggaeth**
PM me if these do not meet the needs of the course. #Hacktoberfest2018

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/css/overflow/index.md [hunk 1]
```diff
@@ -63,3 +63,4 @@ If the content overflows the Y-axis, then that content will be hidden, whilst a
 #### More Information:
 
 CSS-Tricks: <a href='https://css-tricks.com/almanac/properties/o/overflow/' target='_blank' rel='nofollow'>overflow</a>
+W3Schools: <a href='https://www.w3schools.com/css/css_overflow.asp' target='_blank' rel='nofollow'>image source</a>
```
Comment (1andee) [inline guide/english/css/overflow/index.md]: We tend to avoid adding links to w3schools.com as they sometimes have incorrect or incomplete information.

**Review by 1andee — COMMENTED** [review]
Hey @ggaeth!

This is a solid PR, thanks for your effort. I noticed that one of the image links is still broken, for the overflow-x and overflow-y example. Any chance you could squeeze that update into this PR?

**Comment by RandellDawson** [comment]
@1andee @thecodingaviator is going to correct the broken image link on a separate PR, so I am going to merge this one.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**